app/api/v1/cards_api.rb

```diff
@@ -0,0 +1,45 @@
+module V1
+  class Cards_Api < Grape::API
+    HAND_NAME = ["ハイカード","ワンペア","ツーペア","スリー・オブ・ア・カインド","ストレート","フラッシュ","フルハウス","フォー・オブ・ア・カインド","ストレートフラッシュ"]
+
+    resource "card"  do
+      params do
+        requires :cards , type: Array
+      end
+      post"/judge"  do
+        cards =  params[:cards]
+        all_hand = []
+        results = []
+        cards.each do |card|
+           @hands = Cards.new(card)
+           @hands.change_card_to_numbers_and_suits
+           if @hands.valid_size == true && @hands.valid_form != true && @hands.valid_unique == true
+                 @hands.judge_hand
+                 @hand_number = HAND_NAME.find_index(@hands.hand)
+                 all_hand.push(@hand_number)
+           else
+             @hand = nil
+           end
+        end
+         cards.each do |card|
+           @hands = Cards.new(card)
+           @hands.change_card_to_numbers_and_suits
+           if @hands.valid_size == true && @hands.valid_form != true && @hands.valid_unique == true
```

Comment:
you don't have to "write ==true". if you make the method which return ture/false, it is better to add "?" in the end of method name